plugins/github-deployments/src/api/index.ts

```diff
@@ -89,9 +99,9 @@ export type Options = {
 };
 
 const deploymentsQuery = `
-query deployments($owner: String!, $repo: String!, $last: Int) {
+query deployments($owner: String!, $repo: String!, $lastDeployments: Int, $lastStatuses: Int) {
```

Comment:
The `QueryParams` type would need an update as well here right? along with possibly more things